Source/Core/Core/PowerPC/BreakPoints.h

```diff
@@ -69,8 +77,26 @@ class BreakPoints
   void Clear();
   void ClearAllTemporary();
 
+  void SetBreakpointTriggered(bool b, u64 addr = -1, MemCheckCondition cond = MEMCHECK_NONE)
+  {
+      m_breakpointTriggered = b;
```

Comment:
This should be in the implementation file (.cpp), not in the header. By keeping this in the header, you will have to rebuild everything that includes this header whenever it changes.